staging/src/k8s.io/apiserver/pkg/server/dynamiccertificates/tlsconfig.go

```diff
@@ -231,23 +233,26 @@ func (c *DynamicServingCertificateController) syncCerts() error {
 }
 
 // RunOnce runs a single sync step to ensure that we have a valid starting configuration.
-func (c *DynamicServingCertificateController) RunOnce() error {
-	return c.syncCerts()
+func (c *DynamicServingCertificateController) RunOnce(ctx context.Context) error {
+	return c.syncCerts(ctx)
 }
 
 // Run starts the kube-apiserver and blocks until stopCh is closed.
-func (c *DynamicServingCertificateController) Run(workers int, stopCh <-chan struct{}) {
+func (c *DynamicServingCertificateController) Run(ctx context.Context, workers int, stopCh <-chan struct{}) {
```

Comment:
Same here.

I think @deads2k said that these controllers are considered Kubernetes-internal, so it might be okay to change this API.